src/domain/Chat/Chat.module.scss

```
@@ -58,12 +59,38 @@
+    &__nickname {
+      @include m.text-style(xs, 700);
+      color: f.color(gray-600);
+      padding-left: 4px;
+    }
+
+    &__system-banner {
+      padding: 4px 12px;
+      border-radius: 12px;
+      background: rgba(f.color(gray-200), 0.6);
+      color: f.color(gray-600);
+      @include m.text-style(xs, 500);
```

Stylelint (17.2.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)